### ISSUE TITLE
Standalone: segfault on X-Men LE (VPW Version)

### ISSUE THREAD

**Issue by toccata10** [vpinball/vpinball#1152]
**Describe the bug**
After a few seconds of gameplay, the game freezes, the gamepad rumbles continuously and it quits after a few seconds.

**To Reproduce**
1. Launch standalone on Linux
2. start playing X-Men LE Vpw version
3. After a few seconds of gameplay, the game freezes, the gamepad rumbles continuously and it quits after a few seconds.

If the bug appears when playing a specific table, give a link to a place where the exact version of the table file can be downloaded.

**Expected behavior**
Have a playable table.

**Crash dump**
If it's about a crash, please attach `crash.dmp` as well.

**Versions & Settings**
 - VPX standalone
 - Linux version

**Additional context**
[vpinball.log](https://github.com/vpinball/vpinball/files/13396642/vpinball.log)

**Comment by jsm174** [comment]
I would need to know exact table so I can reproduce.

**Comment by toccata10** [comment]
Of course: it's the latest available version from [vpuniverse] (https://vpuniverse.com/files/file/12253-x-men-le-stern-2012-vpw-mod/) (with unzips in "X-Men LE (Stern 2012) VPW v1.0.vpx")
It crashes on a batocera 39 beta and on a compiled vpx-standalone version on Archlinux.

**Comment by jsm174** [comment]
This seems to be the `gBOT` vs `getballs` in `CoRTracker` patch required for Standalone. I've messaged the team on Discord, so hopefully in a future release it will be updated.

If you ever see it crash immediately after the ball is hit by a flipper, it's usually this.

Here is a patch in the mean time:

https://github.com/jsm174/vpx-standalone-scripts/tree/master/X-Men%20LE%20(Stern%202012)%20VPW%20v1.0

(I played this on my Batocera cabinet for about 15 minutes with no crashes.)

For documentation purposes, attached is a video of this happening with the debugger enabled in Haunted House:

 



 

https://github.com/vpinball/vpinball/assets/1197137/cc775d0d-c90e-45fe-9792-c812e89fc1da
